.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Feature request
+about: Have an idea? Let's talk!
+---
+
+Before you open an issue, please check if a similar one already exists or has been closed before. Detailed information about the process of contributing can be found in [CONTRIBUTING.md](https://github.com/vazco/uniforms/blob/master/.github/CONTRIBUTING.md).
+
+Let us know what you'd like to see. Do provide us with:
+
+- Some motivation why you'd like to see this in uniforms.
```

Comment:
I think the "Some" is a little out of place here. 

Also there might be a preposition missing? Like `Motivation for` or `on`? Not sure about this though.

```suggestion
- Motivation why you'd like to see this in uniforms.
```

Reply:
I think it's a common scheme in such context. I agree with the comment on _some_.